### PR TITLE
Added ROT13 and Atbash to sympy.crypto

### LINKED ISSUE #16495: Implementing classical ciphers as special cases of already existing ciphers in crypto.py
The file ```/sympy/crypto/crypto.py``` contains implementations of various classical ciphers.

Extending this file to include certain other classical ciphers which are derivatives of the ones already present in the file would be useful.

Such implementations would be just a couple of lines code (excluding comments explaining the working) for each as most of the ```encipher_``` functions which are required already exist.

Examples of possible extensions are :
[ROT13](https://en.wikipedia.org/wiki/ROT13) as an extension of ```encipher_shift()``` (and ```decipher_shift()``` likewise)
[Atbash](https://en.wikipedia.org/wiki/Atbash) as an extension of ```encipher_affine()``` (and ```decipher_affine()``` likewise)

If possible and if this issue is relevant, I would like to be assigned this issue and work on it and extending the ```crypto.py``` file further.

### PR DESCRIPTION
**Pull request by znxftw**
#### References to other Issues or PRs

Fixes #16495 

#### Brief description of what is fixed or changed
- Added ```encipher_rot13``` , ```decipher_rot13```
- Added ```encipher_atbash```, ```decipher_atbash```
- Updated tests and documentation for both.

#### Other comments
- Both encipher and decipher do the same thing, in theory. However I made them different functions because they are intended for different uses even though they are functionally equivalent, and would also maintain code quality and prevent ambiguity in future.

#### Release Notes

* crypto
  * added ```rot13``` and ```atbash``` ciphers

### CODE REVIEW

**Review by sylee957 — APPROVED** [review]

**Comment by oscarbenjamin** [comment]
The spacing around headers in the docstrings is inconsistent. See here [https://github.com/sympy/sympy/wiki/Writing-documentation#docstrings](https://github.com/sympy/sympy/wiki/Writing-documentation#docstrings) and note blank lines before/after headers.

Otherwise looks good.

**Comment by znxftw** [comment]
Adjusted the spaces as requested. Is this fine, @oscarbenjamin ?

File: sympy/crypto/crypto.py [hunk 1]
```diff
@@ -349,6 +398,53 @@ def decipher_affine(msg, key, symbols=None):
     """
     return encipher_affine(msg, key, symbols, _inverse=True)
 
+def encipher_atbash(msg, symbols=None):
+    r"""
+    Enciphers a given ```msg``` into its Atbash ciphertext and returns it.
+
+    Notes
+    =====
+
```
Comment (sylee957) [inline sympy/crypto/crypto.py]: The tests failing. I see some trailing spaces to be removed here

Comment (znxftw) [reply sympy/crypto/crypto.py]: Oops, sorry.My bad, didn't run the test_code_quality.py before commiting. Will fix it immediately. N++ automatically indents trailing spaces with newlines, didn't notice this time.

File: sympy/crypto/crypto.py [hunk 2]
```diff
@@ -259,6 +259,55 @@ def decipher_shift(msg, key, symbols=None):
...
+def decipher_rot13(msg, symbols=None):
+    """
+    Performs the ROT13 decryption on a given plaintext ```msg```.
+
+    Notes
+    =====
+
+    ```decipher_rot13``` is equivalent to ```encipher_rot13``` as both
+    ```decipher_shift``` with a key of 13 and ```encipher_shift``` key with a
```
Comment (sidhantnagpal) [inline sympy/crypto/crypto.py]: These should use double back-ticks (\`\`) to be rendered as code in Sphinx. Current render can be seen at https://docs.sympy.org/dev/modules/crypto.html#sympy.crypto.crypto.encipher_rot13.

**Pull request merged**